content/backend/graphql-python/1-getting-started.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Getting Started
+description: Setting up a Django Graphene Project
+---
+
+### What are you going to build?
+In this tutorial, you are going to create a [Hackernews](https://news.ycombinator.com) clone using Django and Graphene. The platform will have the following features:
+
+* List of Users and Links
+* Users creation and authentication
+* Users can create Links and vote on them
+
+### Creating your local environment
+Before starting, make sure you have Python 3.6 installed. If not, you can download it [here](https://www.python.org/downloads/). The installation is pretty straightforward on any supported operating system.
+
+When working with Python, you will use *Virtual Environment*: a tool for keeping all the project's dependencies isolated from your system – or other projects.
+
+<Instruction>
+
+On the terminal, create a new virtual environment and activate it:
+
+```bash
+python3.6 -m venv venv
+source venv/bin/activate
+```
+
+You'll notice your prompt changed to `(venv) ...`. It means you are *inside* your virtual environment, every Python package will be installed there. If you want to leave it, just type `deactivate`. Make sure to *always* have the virtual environment activated.
+
+</Instruction>
+
+### Installing Django and Graphene 
+The tool used to manage Python packages is `pip`, which should be available with your Python installation.  With the virtual environment activated, run the following commands:
```

Comment:
"With the virtual environment activated, run the following commands:" --> this part should be inside the `<Instruction>` tags. Otherwise it doesn't render as an instruction block either... Sorry for the flaws in the Markdown parsing 😔